6tisch-roll-enrollment-priority.mkd

```diff
@@ -1,7 +1,7 @@
 ---
 title: Controlling Secure Network Enrollment in RPL networks
 abbrev: join-metric
-docname: draft-ietf-roll-enrollment-priority-09
+docname: draft-ietf-roll-enrollment-priority-10
```

Comment:
It's better to let the person publishing bump this number, otherwise one gets spurious conflicts :-)

Reply:
OK. Sorry. :-) Will remember that.